R/install-dev.sh

```diff
@@ -35,12 +35,19 @@ LIB_DIR="$FWDIR/lib"
 mkdir -p $LIB_DIR
 
 pushd $FWDIR > /dev/null
+if [ ! -z "$1" ]
+  then
+    R_HOME="$1/bin"
+   else
+    R_HOME="$(dirname $(which R))"
```

Comment:
Also if using an environment variable then the code for running sparkR would also have to be edited to use the R version from this path.